src/style.css

```diff
@@ -0,0 +1,88 @@
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir.ttc);
+}
```

Comment:
Before submitting your work, always check your work against the checklist
![image](https://github.com/mate-academy/layout_search-bar-airbnb/assets/104684446/e0f2302c-6ef8-4714-9320-922b89dd6f2e)